deploy/aws/README.md

```diff
@@ -126,13 +126,13 @@ by Grafana. The inference server helm chart assumes that Prometheus
 and Grafana are available so this step must be followed even if you
 don't want to use Grafana.
 
-Use the prometheus-operator to install these components. The
+Use the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) to install these components. The
 *serviceMonitorSelectorNilUsesHelmValues* flag is needed so that
 Prometheus can find the inference server metrics in the *example*
 release deployed below.
 
 ```
-$ helm install --name example-metrics --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false stable/prometheus-operator
+$ helm install example-metrics --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false prometheus-community/kube-prometheus-stack
```

Comment:
Can you replace all instances of the following for consistency in using `helm3`?:
- `helm install --name <name>` -> `helm install <name>` 
- `helm delete --purge <name>` -> `helm uninstall <name>`

If you check your `history` from testing this, I believe these are the versions of the commands you should've used. Use of `--name` in [helm install](https://helm.sh/docs/helm/helm_install/) and [helm delete](https://helm.sh/docs/helm/helm_delete/) was removed in `helm3`.

Reply:
Will update. FYI, seems like in helm3, `delete` is marked as alias of `uninstall`, so it is still available but the usage is following `uninstall`